### PR TITLE
Fixing bug where creating from a template file was busted

### LINKED ISSUE #1005: Create from template can throw an error
Using the `nodejs-mongodb.json` template & pasting it in the `Import YAML / JSON` field, I get a console error w/o any feedback on the page:

screen shot 2016-12-07 at 5 07 33 pm

@jwforres @spadgett

### PR DESCRIPTION
**Pull request by rhamilto**
Fixes #1005 

@benjaminapetersen, @spadgett, @jwforres, PTAL

### CODE REVIEW

File: app/scripts/directives/fromFile.js [hunk 1]
```diff
@@ -118,6 +118,7 @@ angular.module("openshiftConsole")
             return;
           }
 
+          $scope.resource = resource;
```
Comment (spadgett) [inline app/scripts/directives/fromFile.js]: Shouldn't need to set this as a scope var, can just use `resource` below when you call `createFromTemplateURL` below

**Comment by spadgett** [comment]
@rhamilto can you sweep the code for other places where this might be a problem (templates or image streams)

File: app/scripts/directives/fromFile.js (hunk 1 above)
Comment (rhamilto) [inline app/scripts/directives/fromFile.js]: Resource is scoped to only this function.  How do I get it out of this function?

**Comment by rhamilto** [comment]
Ok.  I think I got it this time.  Thanks, @jwforres and @spadgett.

> @rhamilto can you sweep the code for other places where this might be a problem (templates or image streams)

Done.  We should be good now.

**Comment by spadgett** [comment]
[merge]

**Pull request merged**